app/src/main/res/values/strings.xml

```diff
@@ -18,17 +18,17 @@
     <string name="error_downloading">Error Downloading %1$s</string>
     <string name="install">Install</string>
     <string name="installing_app">Installing %1$s</string>
-    <string name="installation_aborted">Operation failed because the user aborted the installation.</string>
-    <string name="installation_blocked">Operation failed because the user blocked the installation.</string>
-    <string name="installation_downgrade">Operation failed because the user tried to downgrade the package. Do you have stock YouTube Installed?</string>
-    <string name="installation_conflict">Operation failed because the app conflicts with an already installed app.</string>
-    <string name="installation_failed">Operation failed for an unknown reason, please join our Telegram or Discord for support.</string>
-    <string name="installation_incompatible">Operation failed because the app is incompatible with your device.</string>
-    <string name="installation_invalid">Operation failed because the apk files are invalid.</string>
-    <string name="installation_signature">Operation failed because the user didn\'t disable apk signature verification</string>
-    <string name="installation_miui">An unexpected error occurred while installing Vanced. Have you disabled MIUI Optimisations?</string>
-    <string name="installation_storage">Operation failed due to a storage error.</string>
-    <string name="installation_wait">Please wait until the installation finishes</string>
+    <string name="installation_aborted">Installation failed because user aborted the installation.</string>
+    <string name="installation_blocked">Installation failed because user blocked the installation.</string>
+    <string name="installation_downgrade">Installation failed because user tried to downgrade the package. Uninstall updates from stock YouTube app then try again.</string>
+    <string name="installation_conflict">Installation failed because the app conflicts with an already installed app. Uninstall the current version of Vanced then try again.</string>
+    <string name="installation_failed">Installation failed for unknown reasons, join our Telegram or Discord for futher support.</string>
+    <string name="installation_incompatible">Installation failed because the installation file is incompatible with your device. Choose another architecture for installation.</string>
```

Comment:
App chooses architecture automatically, but if something fails it should just tell the error, user can't choose arch manually

Reply:
So it is impossible for users to fix this issue?
Or is there a fix?

Reply:
This issue will *never* occur, error handling is just a fallback, just in case something goes wrong (which won't happen). So it's better to just show error instead of telling people what to do, especially when there's nothing a user can do

Reply:
This issue will never occur + just in case something goes wrong + which won't happen.
Can't we just remove this from the strings?

Reply:
It's a fallback measure

Reply:
Okay. If this ever happens (which won't happen), can the users somehow delete the downloaded files then install the app (in case the app downloaded the wrong file but will download the right one on the second pick).

Reply:
There's a button in settings which deletes all downloaded files

Reply:
Oops. I totally forgot its existence.